privateca/snippets/conftest.py

```diff
@@ -31,6 +33,48 @@
 CA_DURATION = 1000000
 
 
+def delete_ca(ca_pool_name: str) -> None:
+    client = privateca_v1.CertificateAuthorityServiceClient()
+    for ca in client.list_certificate_authorities(parent=ca_pool_name):
+        # Check if the CA is enabled.
+        ca_state = client.get_certificate_authority(name=ca.name).state
```

Comment:
Do we need to `get_certificate_authority` after we `list_centrificate_authorities`? Does listing not provide the state in the returned objects?